src/devcenter/azext_devcenter/manual/_validators.py

```diff
@@ -17,3 +18,11 @@ def validate_dev_box_list(namespace):
         raise RequiredArgumentMissingError(
             "--user-id is required when using --project-name/--project."
         )
+
+
+def validate_time(namespace):
```

Comment:
Don't think you regenerated the whl file based off the commit history for these changes (adding the validator)